ext/build.sh

```diff
@@ -44,3 +44,22 @@ cmake .
 make
 make install
 popd
+
+# GPS Parser
+pushd /tmp
+git clone https://github.com/ckgt/NemaTode.git
+cd NemaTode
+cmake .
+make
+make install
+popd
+
+# SNMP library
+pushd /tmp
+wget http://sourceforge.net/projects/net-snmp/files/net-snmp/5.9.1/net-snmp-5.9.1.tar.gz
+tar -xvzf net-snmp-5.9.1.tar.gz
+cd net-snmp-5.9.1
+./configure --with-default-snmp-version="3" --with-sys-contact="@@no.where" --with-sys-location="Unknown" --with-logfile="/var/log/snmpd.log" --with-persistent-directory="/var/net-snmp"
+make
+make install
+popd
```

Comment:
Lets make sure that if we need to use this version we remove the package install in the dependencies script.  We should not have 2 versions of the SNMP library installed as that will create confusion.